sound/soc/sof/intel/pci-mtl.c

```diff
@@ -40,6 +40,7 @@ static const struct sof_dev_desc mtl_desc = {
 	.ipc_supported_mask	= BIT(SOF_IPC_TYPE_4),
 	.ipc_default		= SOF_IPC_TYPE_4,
 	.dspless_mode_supported	= true,		/* Only supported for HDaudio */
+	.on_demand_dsp_boot	= true,
```

Comment:
Should we start with a incremental deployment before we expand to all platforms?